src/_includes/how_to/get_connected_plan_table.html

```diff
@@ -2,7 +2,7 @@
   <tr>
     <th scope="col" class="border-top align-middle table-dark-blue border-dark-blue text-white table-bordered">1 GB</th>
     <td class="border-top border-dark-blue table-bordered">
-      <li class="mx-4">GTFS-RT <em>or</em> 1 fare payment validator</li>
+      <li class="mx-4">GTFS-RT</li>
```

Comment:
@mrose914 Please confirm this change.

This is what was requested on the changelog doc:

<img width="971" alt="image" src="https://github.com/cal-itp/mobility-marketplace/assets/3673236/acc43271-cc97-4637-8dc3-46b9ec0c1910">

This is what's on production:

<img width="980" alt="image" src="https://github.com/cal-itp/mobility-marketplace/assets/3673236/3ef90eb8-bde5-4291-a1cf-ac379d9b1d21">

https://www.camobilitymarketplace.org/how-to/get-connected#choosing-how-much-data